include/mbedtls/cipher.h

```diff
@@ -467,8 +470,8 @@ int mbedtls_cipher_setup( mbedtls_cipher_context_t *ctx,
  * \param cipher_info   The cipher to use.
  * \param taglen        For AEAD ciphers, the length in bytes of the
  *                      authentication tag to use. Subsequent uses of
- *                      mbedtls_cipher_auth_encrypt_ext() or
- *                      mbedtls_cipher_auth_decrypt_ext() must provide
+ *                      mbedtls_cipher_auth_encrypt() or
+ *                      mbedtls_cipher_auth_decrypt() must provide
```

Comment:
This undoes a change that was done in #4445 - this is probably unintentional and needs to be fixed.